lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -46,13 +46,7 @@
         </button>
     <% } else {%>
         <div class="no-action-message">
-            <%- gettext('Coming Soon') %>
-        </div>
-        <div class="enrollment-opens">
-            <%- gettext('Enrollment Opens on') %>
-            <span class="enrollment-open-date">
-                <%- enrollment_open_date %>
-            </span>
+            <%- gettext('Not Currently Available') %>
```

Comment:
I realized something while implementing this change. Given the data we're working with, a learner is actually never going to end up in a situation where they would see a "Coming Soon" message. In the system we're working in today, course enrollment start and end dates are always left blank, which means that as soon as a course is added to the program, it will be available for enrollment. The "Not Currently Available" case is still possible when all course runs have ended. I need to catch up with Lauren on this, but I think the current state of this PR is what we're going to end up wanting to go with.